deployment/job.nomad.hcl

```diff
@@ -29,7 +33,9 @@ job "nox" {
     count = var.replicas
 
     update {
-      max_parallel = 4
+      max_parallel      = 6
+      healthy_deadline  = "10m"
```

Comment:
why 10 mins?

Reply:
Hoping that is more than enough for all nox nodes to be up and healthy. Default is 5 minutes which is now enough right now